site/styles/_availability-tags.scss

```diff
@@ -64,16 +64,16 @@ s72-availability-label {
   }
 
   .availability-bg.live {
-    color: var(--body-color, rgb(255, 255, 255));
     background-color: rgba(var(--body-bg-rgb, 0, 0, 0), 0.7);
+    color: var(--body-color, rgb(255, 255, 255));
```

Comment:
Is this linter moving this into alphabetical order

Reply:
Yup!